firmware/quadruna/VC/test/test_stateMachine.cpp

```diff
@@ -223,3 +230,50 @@ TEST_F(VCStateMachineTest, drive_to_init_state_on_CRIT_fault)
 
     TestFaultBlocksDrive(set_fault, clear_fault);
 }
+
+TEST_F(VCStateMachineTest, drive_to_init_inverter_fault)
+{
+    app_canRx_CRIT_StartSwitch_update(SWITCH_ON);
```

Comment:
Can we make going to drive state a single reusable function, and make a test to make sure that function is working